### ISSUE TITLE
Error with makeFilePathAbsolute

### ISSUE THREAD

**Issue by nitedani** [brillout/vite-plugin-ssr#703]
### Description

Environment:
Vite 3
Windows 11 WSL2 docker node:18.12-alpine3.16

Happened after updating from 0.4.77 to 0.4.93
pinning 0.4.77 is working fine

### Error Message + Error Stack

```shell
> vite -c config/vavite.config.ts
error when starting dev server:
Error: [vite-plugin-ssr@0.4.93][Bug] You stumbled upon a bug in vite-plugin-ssr's source code. Reach out at https://github.com/brillout/vite-plugin-ssr/issues/new or https://discord.com/invite/qTq92FQzKb and include this error stack (the error stack is usually enough to fix the problem). A maintainer will fix the bug (usually under 24 hours). Don't hesitate to reach out as it makes vite-plugin-ssr more robust.
    at makeFilePathAbsolute (/src/node_modules/.pnpm/vite-plugin-ssr@0.4.93_ima47bflrjqvy2qu33dh2efdbq/node_modules/vite-plugin-ssr/dist/cjs/utils/makeFilePathAbsolute.js:20:25)
    at /src/node_modules/.pnpm/vite-plugin-ssr@0.4.93_ima47bflrjqvy2qu33dh2efdbq/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/devConfig/determineOptimizeDepsEntries.js:25:60
    at Array.forEach (<anonymous>)
    at determineOptimizeDepsEntries (/src/node_modules/.pnpm/vite-plugin-ssr@0.4.93_ima47bflrjqvy2qu33dh2efdbq/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/devConfig/determineOptimizeDepsEntries.js:24:19)
    at async configResolved (/src/node_modules/.pnpm/vite-plugin-ssr@0.4.93_ima47bflrjqvy2qu33dh2efdbq/node_modules/vite-plugin-ssr/dist/cjs/node/plugin/plugins/devConfig.js:49:48)
    at async Promise.all (index 9)
    at async resolveConfig (file:///src/node_modules/.pnpm/vite@3.2.5_@types+node@18.15.2/node_modules/vite/dist/node/chunks/dep-5605cfa4.js:62878:5)
    at async createServer (file:///src/node_modules/.pnpm/vite@3.2.5_@types+node@18.15.2/node_modules/vite/dist/node/chunks/dep-5605cfa4.js:61943:20)
    at async CAC.<anonymous> (file:///src/node_modules/.pnpm/vite@3.2.5_@types+node@18.15.2/node_modules/vite/dist/node/cli.js:707:24)
```
```

**Comment by brillout** [comment]
I See what’s going on. I’ll Release a fix.

**Comment by brillout** [comment]
Fix released in `0.4.94`.
